src/components/ExitButton.tsx

```diff
@@ -0,0 +1,57 @@
+import { useState } from "react";
+import { dopplerExternalUrls } from "../abstractions/configuration";
+
+interface ExitButtonProps {
+  externalUrls: dopplerExternalUrls;
+}
+
+export const ExitButton = (props: ExitButtonProps) => {
```

Comment:
What do you think about converting this component in a general dropdown? 

```suggestion
export const DropdownButton = (props: ExitButtonProps) => {
```

An alternative is to move inside this exit button all the behavior and read the configuration here.

What is the responsibility that you want for this component?

Reply:
I think this component should receive some other props like button text, items text and urls and should render all this with dropdown library components, and of course, redirect to each item url.

Reply:
Done! Marking this conversation as solved.